dulwich/tests/test_index.py

```diff
@@ -369,7 +369,10 @@ def test_nonempty(self):
                 sorted(os.listdir(os.path.join(repo.path, 'c'))))
 
     def test_norewrite(self):
-        sync = getattr(os, 'sync', lambda: os.system('sync'))
+        if sys.platform != 'win32':
```

Comment:
Hmm, i don't think so, according to https://docs.python.org/3/library/os.html:

```
os.sync()
Force write of everything to disk.

Availability: Unix.

New in version 3.3.

```

I'll go with the approach of skipping the test on windows .. I've done some cursory research (googling it) and it seems sync-ing system-wide isn't something one easily does on windows.